classes/manager.php

```diff
@@ -263,6 +263,14 @@ public static function set_pass_state() {
             $event = \tool_mfa\event\user_passed_mfa::user_passed_mfa_event($USER);
             $event->trigger();
         }
+
+        // Fire specific factor pass state actions.
+        $pluginsfunction = get_plugins_with_function('mfa_post_pass_state');
```

Comment:
1) why not just a method in the factor and not a new callback hook?

2) Should this be moved above inside the if so it doesn't fire duplicates?